test/unit/sanitiser/_geo_reverse.js

```diff
@@ -50,7 +50,7 @@ module.exports.tests.sanitize_boundary_country = function(test, common) {
     // t.equals(clean['boundary.circle.radius'], 12.121212, 'should be set to point.lat')
     t.deepEquals(errorsAndWarnings, {
       errors: [],
-      warnings: ['boundary.circle is currently unsupported']
+      warnings: []
```

Comment:
sorry, ignore the first comment. i was reading this wrong.

however, the name of the test should be updated to reflect what the new expectation is.